nntrainer/include/tensor.h

```diff
@@ -580,6 +579,7 @@ class Tensor {
     return strides;
   }
 
+  // TODO: move to private
```

Comment:
There is a test with this function. I guess the test can be removed because it is tested through `opeator_i`

Reply:
I am currently disabling the test than removing it. We can think about removing the test or making the test class a friend class for testing purposes wrapped with `#if #endif`.

Reply:
Ohh that test is already DISABLED and its not an-automated test. Removing it.